package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "wallet-ui",
+  "version": "0.1.0",
+  "description": "Wallet UI",
```

Comment:
Please expand this description a little bit

Reply:
Done